UnityProject/Assets/Scripts/Objects/ObjectContainer.cs

```diff
@@ -158,10 +158,11 @@ public void StoreObject(GameObject obj, Vector3 offset = new Vector3())
 				Loggy.LogError("[ObjectContainer/StoreObject] - HEY SHITASS, DON'T TRY ADDING NULL OBJECTS.");
 				return;
 			}
-			storedObjects.Add(obj, offset);
-			OnObjectStored?.Invoke(obj);
 			if (obj.TryGetComponent<UniversalObjectPhysics>(out var objectPhysics))
 			{
+				if (objectPhysics is MovementSynchronisation { CanMoveThroughObstructions: true }) return;
```

Comment:
use  Intangible bool instead since that specifies whether or not it can interact with the physical world